### DISCUSSION TITLE
Persistence in NRules

### DISCUSSION THREAD

**Discussion by RekikMohamed** [NRules/NRules#366] [Q&A] [answered]
I want to implement the following condition with persistence.
I have x, y, z as facts.
And my condition is x>1 && y>2 && z >0. I want the result of that conditions to stay true for 5 minutes straight. If it does, it will trigger the rule. If it changes to false before the 5 minutes, the rule does not get triggered.
Does NRules offer this functionality?

**Comment by snikolayev** [comment, marked as answer]
Not sure what this has to do with persistence, but no, NRules is not aware of the clock or the passage of time
